07_deploy_masters.sh

```diff
@@ -69,9 +69,15 @@ done
 echo "Master nodes up, you can ssh to the following IPs with core@<IP>"
 sudo virsh net-dhcp-leases baremetal
 
+# Wait for nodes to become ready
+NUM_LEASES=$(oc --config ocp/auth/kubeconfig get nodes | wc -l)
```

Comment:
We should probably rename the variable to NUM_NODES or something?

Reply:
Done